tools/awslint/bin/awslint.ts

```diff
@@ -265,9 +265,9 @@ function mergeOptions(dest: any, pkg?: any) {
 }
 
 async function shell(command: string) {
-  const child = child_process.spawn(command, undefined, { stdio: [ 'inherit', 'inherit', 'inherit' ]});
+  const child = child_process.spawn(command, [], { stdio: [ 'inherit', 'inherit', 'inherit' ]});
   return new Promise((ok, ko) => {
-    child.once('exit', status => {
+    child.once('exit', (status: any) => {
```

Comment:
I believe these changes are because of upgrades to `@types/fs-extra` and/or TypeScript.

Reply:
Might be this code catching a different `@types/node` version due to local dependencies bringing in a `>= 10` or `>= 12` version here... I've had this happen in other places last time.

Reply:
How did you deal with it?